src/grammar/__snapshots__/grammar.spec.ts.snap

```diff
@@ -254,6 +254,34 @@ Line 2, col 31:
 "
 `;
 
+exports[`grammar should fail case-27 1`] = `
+"<unknown>:1:19: Empty inherited traits list should not have a dangling comma.
```

Comment:
We should disallow empty trait lists entirely. We allow it now, but it's a bug, see #247, so let's fix it here.

Reply:
Agreed